src/ux/confirmInformation/ConfirmInformation.tsx

```diff
@@ -0,0 +1,111 @@
+// eslint-disable-next-line
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { SD } from '../../lib/sdr'
+import { Header2, SmallSpan, Span, Span2 } from '../../ui/shared/Typography'
+import { Button, ButtonSecondary } from '../../ui/shared/Button'
+import { credentialValueToText } from '../../vc-json-schema-adapter'
+import Loading from '../../ui/shared/Loading'
+import { Trans } from 'react-i18next'
+import i18next from 'i18next'
+import { getChainName } from '../../adapters'
+import { IProviderUserOptions } from 'web3modal'
+import Checkbox from '../../ui/shared/Checkbox'
+
+const DONT_SHOW_AGAIN_KEY = 'RLogin:DontShowAgain'
+
+interface ConfirmInformationProps {
+  chainId: number | undefined
+  address: string | undefined
+  sd: SD | undefined
+  providerUserOption: IProviderUserOptions
+  onConfirm: () => void
+  onCancel: () => void
+}
+
+export function ConfirmInformation ({ chainId, address, providerUserOption, sd, onConfirm, onCancel }: ConfirmInformationProps) {
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+  const [dontShowAgainSelected, setDontShowAgainSelected] = useState<boolean>(false)
+  const data = sd ? Object.assign({}, sd.credentials, sd.claims) : {}
+
+  const handleSubmit = () => {
+    setIsLoading(true)
+    onConfirm()
+  }
+
+  useEffect(() => {
+    const DONT_SHOW_AGAIN_DEFAULT: boolean =
+      localStorage.getItem(DONT_SHOW_AGAIN_KEY) ? JSON.parse(localStorage.getItem(DONT_SHOW_AGAIN_KEY)!) : false
+
+    if (DONT_SHOW_AGAIN_DEFAULT) {
+      handleSubmit()
+    }
+  }, [])
+
+  return !isLoading
+    ? <>
+      <Header2><Trans>Information</Trans></Header2>
+      <CenterContent>
+        <ProviderLogo>
+          <img src={providerUserOption.logo} alt={providerUserOption.name} />
+        </ProviderLogo>
+      </CenterContent>
+
+      <div style={{ display: 'flex', padding: '50px 0', justifyContent: 'center', gap: '30px' }}>
```

Comment:
Are we mixing and matching styledComponents with inline CSS? It would be best to stick with the styled components since it will output the CSS and not clutter up the outputted HTML.